src/charm.py

```diff
@@ -73,10 +79,16 @@ def _validate_config(self):
             return False
         return True
 
-    @property
-    def external_hostname(self):
-        """Check if hostname has been configured. If not, generate one."""
-        return self.config["external-hostname"] or "{}.juju".format(self.app.name)
+    def _configure_ingress(self, event):
+        """Sends data for ingress relation."""
+        if self.interfaces["ingress"]:
+            self.interfaces["ingress"].send_data(
+                {
+                    "prefix": "/volumes/",
```

Comment:
Can you add the following field?
`"rewrite": "/"`
Also, the `/` at the end shouldn't be needed.